platform/ios/CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Fixed a crash or console spew when MGLMapView is initialized with a frame smaller than 64 points wide by 64 points tall. ([#8562](https://github.com/mapbox/mapbox-gl-native/pull/8562))
 * The error passed into `-[MGLMapViewDelegate mapViewDidFailLoadingMap:withError:]` now includes a more specific description and failure reason. ([#8418](https://github.com/mapbox/mapbox-gl-native/pull/8418))
 * Fixed an issue rendering polylines that contain duplicate vertices. ([#8808](https://github.com/mapbox/mapbox-gl-native/pull/8808))
+* Fixed static framework size by using correct llvm flags on release scheme for strip command. ([#9021](https://github.com/mapbox/mapbox-gl-native/issues/9021))
```

Comment:
If this change should go into iOS SDK v3.6.0, this PR would need to be retargeted and rebased onto the release-ios-v3.6.0-android-v5.1.0 branch. Otherwise, this note should be moved up to the “master” section, which will go into iOS SDK v3.7.0.

/cc @boundsj